lib/remote-proxy.js

```diff
@@ -55,15 +57,8 @@ RemoteProxy.wrapRemoteMethod = (instance, methodName) => {
 //   methodName - name of a remote method
 //
 function remoteMethodWrapper(methodName, ...args) {
-  let callback = args[args.length - 1];
-  args = Array.prototype.slice.call(args, 0, -1);
-
-  if (typeof(callback) !== 'function') {
-    args.push(callback);
-    callback = null;
-  }
-
-  this._connection.callMethod(this._interfaceName, methodName, args, callback);
+  const cb = common.cbExtract(args);
```

Comment:
Can we please call it `callback`? I'd prefer to use descriptive names when possible.